pkg/app/piped/planner/kubernetes/kubernetes.go

```diff
@@ -97,6 +97,18 @@ func (p *Planner) Plan(ctx context.Context, in planner.Input) (out planner.Outpu
 		out.Version = version
 	}
 
+	if versions, e := determineVersions(newManifests); e != nil {
+		in.Logger.Error("unable to determine versions", zap.Error(e))
+		out.Versions = []*model.ArtifactVersion{
+			{
+				Kind:    model.ArtifactVersion_CONTAINER_IMAGE,
```

Comment:
nit: We can use this kind https://github.com/pipe-cd/pipecd/blob/0c6a1b3f464b659e83cc5f1aa280fa8f914e4c92/pkg/model/application.proto#L90